react/features/stream-effects/blur/index.js

```diff
@@ -11,6 +11,17 @@ const models = {
     '144': 'libs/segm_full_v679.tflite'
 };
 
+const segmentationDimensions = {
+    'version96': {
```

Comment:
on the other object we use the numbers directly as the keys, let's pick one and be consistent. Let's say `modelXX`